IRremoteESP8266.cpp

```diff
@@ -450,6 +450,45 @@ void ICACHE_FLASH_ATTR IRsend::sendDenon (unsigned long data,  int nbits) {
   ledOff();
 }
 
+// Gree, from https://github.com/ToniA/arduino-heatpumpir/blob/master/GreeHeatpumpIR.cpp
+void ICACHE_FLASH_ATTR IRsend::sendGree(unsigned char data[]) {
```

Comment:
As it's 8 x 8-bit chars, you technically could fit it in a unsigned long long/uint64_t if you wanted.